core/toolbox/category.js

```diff
@@ -129,6 +126,8 @@ class ToolboxCategory extends ToolboxItem {
      */
     this.flyoutItems_ = [];
 
+    if (categoryDef === ToolboxCategory.SENTINEL) return;
```

Comment:
I think my preference is to call `parseContents_` in the init method. I think someone not calling init after creating a toolbox item is probably rare and goes against what we would generally expect. The toolbox item would already not work correctly if they do not call init, so the fact that it would break further if they don't call init seems ok to me. Definitely still a breaking change, but I think an acceptable one.

Reply:
Coolio. I'm going to attempt to implement calling `parseContents_` from `init`, because I agree that atm that seems like the best option.